cyvcf2/tests/test_reader.py

```diff
@@ -1316,3 +1316,14 @@ def test_issue236():
 
         assert res[0] == res[1]
         assert len(res[0]) > 0
+
+
+def test_issue17_no_gt():
+    vcf = VCF(os.path.join(HERE, "test-no-genotypes.vcf"))
+    for v in vcf:
```

Comment:
can you use: `with pytest.raises(Exception):` as a more direct way to test this?